### ISSUE TITLE
Generator loses accuracy after a while

### ISSUE THREAD

**Issue by Schobbish** [Schobbish/pigen#4]
In very large calculations, the π returned is less than the actual value. (this was observed only once.) I think this is a problem with the equation used to make the maxK value, since a higher maxK would’ve made the generated π bigger and more accurate. Plus, the inaccuracy started at the very end. More testing as well as a different equation model (the correlation is probably not linear) that errs more on the side of error (deviates away from the actual maxK value needed) would probably solve this. 

### What was run:
`./superpigen -d 126034`

### Output:
The correct π, except for the last 144 digits (five extra digits have been included for context). 
`...6630391432303785641222548279778632100871292788080118178972661856291837186564700028179063426613668938414097112345989258933347005294407031798302873909`

### What should've happened:
This is what should've been outputted instead of the numbers above:
`...6630430784350785928104478492663952652398718644173380085681692321347429754583269402161253332837900960648627785494126679513674045877416945596140762657`

**Comment by Schobbish** [comment]
Commit 12ceebc4bc1bcfd4cbf3d77880d756dd800b67cd fixed this somewhat, but it still got the last two digits wrong (45 instead of 57) when `superpigen -d 126034` was run again. The maxK used this time was 9003 (last time it was 8877).

Another term (ax²) is probably needed in the equation to generate the maxK because it doesn't seem to be linear. This is unfortunate because that can throw the equation wildly off for very high places. More testing is needed to determine the pattern. I don't want to simply lower the ax term because based on what's happening here it won't fix it entirely.
